qa/rpc-tests/walletbackup.py

```diff
@@ -133,9 +133,21 @@ def run_test(self):
         for i in range(5):
             self.do_one_round()
 
-        # Generate 101 more blocks, so any fees paid mature
-        self.nodes[3].generate(101)
-        self.sync_all()
+        print "Generate 101 more blocks, so any fees paid mature"
+        #separating the block creation in this step is to get around issue #6554 which happens on Windows
+        if os.getenv('EXEEXT', "") == '.exe': 
+            print "WARNING: mining smaller numbers of blocks in this step to workaround issue #6554"
+            self.nodes[3].generate(25)
+            self.sync_all()
+            self.nodes[3].generate(25)
+            self.sync_all()
+            self.nodes[3].generate(25)
+            self.sync_all()
+            self.nodes[3].generate(26)
+            self.sync_all()
+        else:
+            self.nodes[3].generate(101)
+            self.sync_all()
```

Comment:
This change feels wrong. Platform specific code in tests and Github issue references in the source code should be avoided.
Is there no way to fix the root cause of this issue? If we can't sync >25 blocks on windows then there is a issue which needs fixing.